freqtrade/misc.py

```diff
@@ -218,6 +218,7 @@ def build_subcommands(parser: argparse.ArgumentParser) -> None:
     'type': 'object',
     'properties': {
         'max_open_trades': {'type': 'integer', 'minimum': 1},
+        'ticker_interval': {'type': 'integer', 'minimum':1, 'maximum':1440},
```

Comment:
I think we must use an enum here instead of an Int.
Because people will enter weird numbers, like '74'. What will happen in this case?